build/README.md

```diff
@@ -171,12 +171,12 @@ a PR to that effect would be greatly appreciated.
 1.  Use [`make-certs.py` script](./make-certs.py) to create certificates for
     the new CockroachDB cluster:
 
-        ./make-certs.py --cluster $CLUSTER_CONTEXT --namespace $NAMESPACE
+        python3 ./make-certs.py --cluster $CLUSTCONTEXT --namespace $NAMESPACE
```

Comment:
To be consistent with other script usage in this repository, could we instead change the first line of `make-certs.py` to be the following?

```python
#!/usr/bin/env python3
```

Reply:
Good idea. Will do.